### DISCUSSION TITLE
Never alert

### DISCUSSION THREAD

**Discussion by ztardik** [seemoo-lab/AirGuard#122] [Q&A]
So, I'm carrying an AirTag I'm my backpack for a month now. The app is running 24/7 and the phone is almost always near the tag. The app never alerted, probably because ithe tag changes it's MAC address all the time.

**Comment by Sn0wfreezeDev** [comment]
When you have the iOS device nearby as well, then the AirTag is not in tracking mode so it won't be detected. 
The security risk of an AirTag tracking you only exists when the owner's iOS device is not nearby.
